CHANGES

```diff
@@ -37,6 +37,18 @@ Notable Bug Fixes
 - Fix an issue in which the loop vectorizer produced code with invalid
   memory reads (issue #757).
 
+Security
+--------
+
+Added a script to automatically apply hardening flags present in some modern
+compilers. Applying these flags may worsen performance of the compilation a
```

Comment:
I still think this is long enough to go to the user manual.  Similarly to this:  https://github.com/pocl/pocl/pull/859/commits/124a04f957e69fdfdc29eb1aa6f9b7305ecf5402 

In CHANGES a sentence like "Added a build parameter HARDENING_ENABLE that applies hardening flags present in some modern compilers to produce a more secure libpocl.so with the trade-off in performance." suffices.

Reply:
done.